subsys/bluetooth/host/hci_core.c

```diff
@@ -3334,7 +3334,7 @@ static void hci_vs_init(void)
 	LOG_INF("Firmware: %s (0x%02x) Version %u.%u Build %u", vs_fw_variant(rp.info->fw_variant),
 		rp.info->fw_variant, rp.info->fw_version, sys_le16_to_cpu(rp.info->fw_revision),
 		sys_le32_to_cpu(rp.info->fw_build));
-#endif /* CONFIG_BT_DEBUG */
+#endif /* CONFIG_BT_HCI_CORE_LOG_LEVEL_INF */
```

Comment:
Correct me if I am wrong, but isn't the no reason to guard calls to `LOG_INF` with `CONFIG_BT_HCI_CORE_LOG_LEVEL_INF`? 

Essentially the only thing we are guarding here is `rp.info = (void *)rsp->data;`, which I would expect the compiler to handle perfectly fine if `INF` logging was disabled. 

To me at least, it looks like we can just remove the guard altogether.